docs/SandboxExtension.md

```diff
@@ -27,6 +27,7 @@ nearby stops generated by routing via OSM data.
 - [Vehicle Parking](sandbox/VehicleParking.md) - Vehicle Parking updaters
 - [Vehicle-to-stop heuristics](sandbox/VehicleToStopHeuristics.md) - Speeding up Park+Ride, Bike+Ride and Bike+Transit searches
 - [Travel Time (Isochrone & Surface) API](sandbox/TravelTime.md) - Travel Time API
+- [Travel Time (Isochrone & Surface) API](sandbox/TravelTime.md) - Travel Time API
```

Comment:
Probably ment to write something else here?

Reply:
No, this is intentional. I noticed that those pages don't have menu items so I added them.